pkgs/applications/networking/browsers/chromium/common.nix

```diff
@@ -18,10 +18,12 @@
 
 # optional dependencies
 , libgcrypt ? null # gnomeSupport || cupsSupport
+, libxkbcommon ? null, libdrm ? null, wayland ? null # useOzoneWayland
 
 # package customization
 , enableNaCl ? false
 , enableWideVine ? false
+, useOzoneWayland ? false
```

Comment:
possibly should just be "useOzone" since there is an x11 backend for Ozone and I currently have it enabled.

Reply:
That would be less confusing.